benchexec/tools/pinaka.py

```diff
@@ -5,34 +5,40 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-import benchexec.util as util
+from benchexec.tools.sv_benchmarks_util import get_data_model_from_task
 import benchexec.tools.template
 import benchexec.result as result
 
 
-class Tool(benchexec.tools.template.BaseTool):
+class Tool(benchexec.tools.template.BaseTool2):
 
     REQUIRED_PATHS = ["pinaka-wrapper.sh", "pinaka"]
 
-    def executable(self):
-        return util.find_executable("pinaka-wrapper.sh")
+    def executable(self, tool_locator):
+        return tool_locator.find_executable("pinaka-wrapper.sh")
 
     def version(self, executable):
         return self._version_from_tool(executable)
 
     def name(self):
         return "Pinaka"
 
-    def cmdline(self, executable, options, tasks, propertyfile, rlimits):
-        if propertyfile:
-            options = options + ["--propertyfile", propertyfile]
+    def cmdline(self, executable, options, task, rlimits):
+        if task.property_file:
+            options = options + ["--propertyfile", task.property_file]
+        data_model_param = get_data_model_from_task(
+            task, {"ILP32": "--32", "LP64": "--64"}
+        )
+        if data_model_param and data_model_param not in options:
+            options += [data_model_param]
 
-        return [executable] + options + tasks
+        return [executable] + options + list(task.input_files_or_identifier)
 
-    def determine_result(self, returncode, returnsignal, output, isTimeout):
+    def determine_result(self, run):
+        output = run.output._lines
```

Comment:
Please remove all accesses to `_lines` from all of the tool-info modules.

Reply:
fixed in af5bd81bdaf51faa9778931901fcbb1b5f251175